packages/contact-exchange/src/services/DIDCommOOBInvitation.ts

```diff
@@ -25,4 +25,14 @@ interface OutOfBandInvitation {
   attachments?: Array<unknown>;
 }
 
+export enum MessageType {
+  RoutingAccept = 'https://mediator.rootsid.cloud',
```

Comment:
It is really strange to have this value here.

Reply:
Yes i agree, thanks for pointing it out.